operations/helm/charts/alloy/README.md

```diff
@@ -85,6 +85,10 @@ useful if just using the default DaemonSet isn't sufficient.
 | controller.nodeSelector | object | `{}` | nodeSelector to apply to Grafana Alloy pods. |
 | controller.parallelRollout | bool | `true` | Whether to deploy pods in parallel. Only used when controller.type is 'statefulset'. |
 | controller.podAnnotations | object | `{}` | Extra pod annotations to add. |
+| controller.podDisruptionBudget | object | `{"enabled":false,"maxUnavailable":null,"minAvailable":null}` | PodDisruptionBudget configuration. |
```

Comment:
```suggestion
| controller.podDisruptionBudget | object | `{}` | PodDisruptionBudget configuration. |
```
This is redundant and can be removed

Reply:
I will be applying the changes you mentioned soon

Reply:
Hey, When I removed this one, helm generate docs exits as you saw in the action / ci part